webapp/src/components/DatasetWarnings/DatasetDistribution.tsx

```diff
@@ -4,6 +4,22 @@ import MetricsDataGrid from "components/DatasetWarnings/MetricsDataGrid";
 import { ResponsivePlotWrapper, WarningPlot } from "components/PlotWrapper";
 import React from "react";
 import { DatasetWarningGroup } from "types/api";
+import Description from "components/Description";
+
+interface DescriptionMap {
+  [key: string]: string;
+}
+
+const WARNING_GROUP_DESCRIPTION: DescriptionMap = {
+  GeneralWarnings: "See class distributions differences. ",
+  SyntacticWarnings:
+    "Review the arrangement of utterances between your training and testing data. ",
+} as const;
```

Comment:
Since you specify a type `: DescriptionMap` (or `: Record<string, string>`), the `as const` doesn't do anything.
```suggestion
};
```

Reply:
okay I understood about const assertion in a wrong way. but this article gave a good explanation on the usage https://stackoverflow.com/questions/66993264/what-does-the-as-const-mean-in-typescript-and-what-is-its-use-case.